core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -41,7 +41,7 @@ oppia.controller('Dashboard', [
       $window.location = '/create/' + explorationId;
     };
 
-    $scope.myExplorationsView = 'list';
+    $scope.myExplorationsView = 'card';
```

Comment:
It's ok to have very small PRs -- this helps the small but important fixes get merged and not be blocked on everything else in the PR being ok.